lib/gs-app-list.h

```diff
@@ -32,6 +32,9 @@ G_BEGIN_DECLS
 
 G_DECLARE_FINAL_TYPE (GsAppList, gs_app_list, GS, APP_LIST, GObject)
 
+typedef gboolean (*GsAppListFilterFunc)		(GsApp		*app,
+						 gpointer	 user_data);
```

Comment:
I believe the alignment must be okay outside of GitHub's view, as it also showed like this in the code @smspillaz removed, and shows okay in my terminal.